countries/burundi/pom.xml

```diff
@@ -46,11 +51,47 @@
           </execution>
         </executions>
       </plugin>
+      <!-- hooking the OpenMRS config validation to the integration-test phase -->
+      <plugin>
+        <groupId>org.openmrs.maven.plugins</groupId>
+        <artifactId>openmrs-packager-maven-plugin</artifactId>
+        <executions>
+          <execution>
+            <id>Validate OpenMRS configuration</id>
+            <phase>integration-test</phase>
+            <goals>
+              <goal>validate-configurations</goal>
+            </goals>
+            <configuration>
+              <sourceDir>${project.build.directory}/${project.artifactId}-${project.version}/distro/configs/openmrs/initializer_config</sourceDir>
+            </configuration>
+          </execution>
+        </executions>
+      </plugin>
+
       <plugin>
-        <groupId>org.apache.maven.plugins</groupId>
         <artifactId>maven-resources-plugin</artifactId>
-        <version>3.3.1</version>
         <executions>
+          <execution>
+            <!-- Copy UVL Disto docker compose .txt file -->
+            <id>Copy UVL Disto docker compose .txt file</id>
```

Comment:
Copy the `docker-compose-files.txt` from root folder to build/run/docker/scripts folder